packages/sockethub-platform-irc/index.js

```diff
@@ -576,22 +578,27 @@ IRC.prototype.__connect = function (key, credentials, cb) {
   const is_secure = (typeof credentials.object.secure === 'boolean') ?
     credentials.object.secure : true;
 
-  const module_creds = {
+  const module_options = {
     username: credentials.object.nick,
```

Comment:
I think if an `object.username` is set, it should be used instead of the nickname. Nicknames can be different from usernames.

Reply:
Makes sense, added that as a possible property in the credentials.